app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java

```diff
@@ -0,0 +1,122 @@
+package org.openobservatory.ooniprobe.receiver;
+
```

Comment:
Please add some short documentation for this new package!

Reply:
package level docs not possible without `package-info.java`